src/styles/_toolbox.scss

```diff
@@ -17,6 +28,56 @@
     &-hide {
         width: 0;
     }
+
+    &-list {
+        padding: 0;
+
+        &:hover {
+            .plantingjs-toolbox-item {
+                .plantingjs-toolbox-image {
+                    opacity: .8;
+                }
+
+                &:hover {
+                    .plantingjs-toolbox-image {
+                        opacity: 1;
+                    }
+                }
+            }
+        }
+    }
+
+    &-item {
+        display: block;
+        margin: 0;
+        border-bottom: 1px solid $secondary-color;
+    }
+
+    &-image {
+        display: block;
+        margin: 0 auto;
+        transition: opacity $default-transition;
+    }
+
+    &-container {
+        box-shadow: $default-shadow;
+        height: 100%;
+        overflow: auto;
+    }
+}
+
+.plantingjs-toolboxobject-item {
+    position: relative;
+    margin-bottom: 2px;
+    min-height: 100px;
+    background: #fafafa;
 }
 
+.plantingjs-toolboxobject-prototype {
 
+}
+// .plantingjs-toolboxobject-draggable {
```

Comment:
There's no point to save commented styles. If we are not using them, they can be removed.